engine/pinball_game/object_collision/blue_stage_resolve_collision.asm

```diff
@@ -5929,7 +5929,7 @@ GoToBonusStageTextIds_BlueField:
 	db STAGE_DIGLETT_BONUS
 	db STAGE_SEEL_BONUS
 
-Func_1e8f6: ; 0x1e8f6
+QueueGraphicsDataToLoad: ; 0x1e8f6

```

Comment:
This isn't correct.  It's Func_10aa that should be labeled as QueueGraphicsDataToLoad.  This function, Func_1e8f6, is calling Func_10aa (QueueGraphicsDataToLoad).